src/player.py

```diff
@@ -12,13 +13,13 @@ def __init__(self, state=None):
         self.history = defaultdict(lambda: False)
 
 
-    def save_to(self, filename):
+    def save_to(self, filename: str):
         """Save the inventory, state, stats and history of the player to the file.
         Can be loaded with 'load_player_from' later"""
         pass
 
 
-    def set_state(self, state):
+    def set_state(self, state: State):
```

Comment:
The player store only the key of the state... so it should be a string -> [here](https://github.com/HopsonCommunity/TextAdventure/blob/develop/src/story.py#L32)